### PR TITLE
Add Type definitions for non-npm package ZWJSBridge API

### PR DESCRIPTION
**Pull request by shlyren**
Type definitions for non-npm package ZWJSBridge API - zwjsbridge.js 1.0.1

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

If removing a declaration:
- [ ] If a package was never on Definitely Typed, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

File: types/zwjsbridge/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,505 @@
+// Type definitions for non-npm package ZWJSBridge API - zwjsbridge.js 1.0
+// Project: https://assets.zjzwfw.gov.cn/assets/ZWJSBridge/1.0.1/zwjsbridge.js
+// Definitions by: Yuxiang Ren <https://github.com/shlyren>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.2
+
+
+interface ZWJSBridge {
+	/**
```
Comment (DangerBotOSS) [inline types/zwjsbridge/index.d.ts:9]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/zwjsbridge/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,504 @@
+// Type definitions for non-npm package ZWJSBridge API - zwjsbridge.js 1.0
+// Project: https://assets.zjzwfw.gov.cn/assets/ZWJSBridge/1.0.1/zwjsbridge.js
+// Definitions by: Yuxiang Ren <https://github.com/shlyren>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 4.2
+
+interface ZWJSBridge {
+	/**
```
Comment (DangerBotOSS) [inline types/zwjsbridge/index.d.ts:8]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/zwjsbridge/index.d.ts [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+     */
+    onReady(callBack: () => void): void
+    /////// 缓存 //////
+    /**
+     * 存储数据缓存
+     */
+    setLocalStorage(options: {
+        key: string
+        value: string
+	}): Promise<{
```
Comment (DangerBotOSS) [inline types/zwjsbridge/index.d.ts:20]: - :no_entry_sign: The root .editorconfig style specifies spaces for whitespace. Please use spaces in new or changed types.

File: types/zwjsbridge/tslint.json [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+    "max-line-length": false,
+    "void-return": false,
+    "npm-naming": false,
+    "no-empty-interface": false,
+    "no-consecutive-blank-lines": false,
+    "no-unnecessary-class": false,
+    "no-trailing-whitespace": false,
+    "no-const-enum": false,
+    "no-dead-reference": false
+  }
```
Comment (peterblazejewicz) [inline types/zwjsbridge/tslint.json:18]:
```suggestion
  "extends": "@definitelytyped/dtslint/dt.json"
```
- can you run:
```bash
npm run prettier -- --write ./types/zwjsbridge/*.ts
```
that will remove nearly all of the problems in your code:
https://github.com/DefinitelyTyped/DefinitelyTyped#common-mistakes (formatting)

and, sorry to ask for this, I understand that character is perfectly OK under zh/ja/ko, replace
```ts
。*/
```
with:
```
 */
```
Looks like JSDoc linter fails when comment content ends with this specific character.

File: types/zwjsbridge/tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,16 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": ["es6", "dom"],
```
Comment (peterblazejewicz) [inline types/zwjsbridge/tsconfig.json:4]:
```suggestion
        "lib": ["es6"],
```
your `index.d.ts` does not depend on `dom` lib. The tests are, as you added `console` symbol.
Please replace console with types assertions, or just do not use console:
https://github.com/DefinitelyTyped/DefinitelyTyped#my-package-teststs

**Review by gabritto — CHANGES_REQUESTED** [review]
Please address the comments left by Peter.

**Review by peterblazejewicz — APPROVED** [review]
LGTM!
@shlyren congrats for your first contribution here! Thanks!

**Pull request merged**